schedule/qam/common/mau-extratests-desktop.yaml

```diff
@@ -0,0 +1,31 @@
+---
+name: mau-extratests-desktop
+schedule:
+   - boot/boot_to_desktop
```

Comment:
I just noticed that this file has a weird indentation of 3 spaces, also in line 19.
The recommended number of spaces is two, and I saw that we don't have that in the `.yamllint` file.

Maybe this is something we should add?

Reply:
Thanks for pointing this out!  Modified the file with two spaces.
yes, we can introduce this check in .yamllint file.

Reply:
I checked the existing yaml files in the repo, and this would require to change 150 files as they are all using a non standard indentation.
Do others have an opinion on that?